Build failure on 32 bit architectures

`_build/src/github.com/facebookincubator/go2chef/multilogger.go:26:3: constant 9223372036854775807 overflows int`

See https://bugzilla.redhat.com/show_bug.cgi?id=1912666 for details. This came up while working on packaging go2chef for Fedora in https://bugzilla.redhat.com/show_bug.cgi?id=1912668

--- a/Makefile
+++ b/Makefile
@@ -27,6 +27,10 @@ vet:
 linux:
 	GOOS=linux GOARCH=amd64 scripts/build.sh
 
+.PHONY: linux32
+linux32:
+	GOOS=linux GOARCH=386 scripts/build.sh
+
 .PHONY: darwin
 darwin:
 	GOOS=darwin GOARCH=amd64 scripts/build.sh

--- a/multilogger.go
+++ b/multilogger.go
@@ -6,7 +6,6 @@ package go2chef
 
 import (
 	"fmt"
-	"math"
 	"runtime"
 )
 
@@ -23,7 +22,7 @@ type MultiLogger struct {
 func NewMultiLogger(loggers []Logger) *MultiLogger {
 	return &MultiLogger{
 		loggers: loggers,
-		debug:   math.MaxInt64,
+		debug:   int(^uint(0) >> 1),
 		level:   LogLevelDebug,
 	}
 }

